_docs_integrate/delete-an-attribute.md

```diff
@@ -0,0 +1,127 @@
+---
+# !!! Warning: Do not edit this file; any changes must be replicated in Excel !!!
+permalink: integrate/delete-an-attribute
+published: true
+title: "Delete an Attribute"
+type: scenario
+toc: true
+properties:
+  - id: SC055
+  - category: Handling data deletion
+  - description:
+  - customer: All
+  - component: integrate
+  - level: Advanced
+  - implementation status: OPEN
+  - documentation status: OPEN
+  - published:
+  - link: integrate/delete-an-attribute
+require:
+required_by:
+---
+
+The exact process of deleting a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) depends on the kind of LocalAttribute at hand.
+Besides creating [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for yourself, a common usage of enmeshed is to share them with a peer or to receive shared Attributes from a peer.
+These different cases have different demands that need to be taken into account, wanting to delete an Attribute.
+
+In general, you can only delete data from your own wallet and never from the peer's.
+So, you can delete peer shared Attributes, your copy of own shared Attributes and unshared RepositoryAttributes.
+However, if you want the peer to delete an Attribute you shared with them, you will need to request it, since the peer may have a valid reason to still keep it for a certain amount of time.
+
+## Request the deletion of own Attributes from peer
```

Comment:
I understand your point of view. I think my main problem with the nomenclature is that it seems quite complex to me (Attribute as an umbrella term for IdentityAttribute and RelationshipAttribute, LocalAttribute as an umbrella term for RepositoryAttribute, own shared IdentityAttribute, peer shared IdentityAttribute, own shared RelationshipAttribute, peer shared RelationshipAttribute, own shared Attribute, peer shared Attribute, ThirdPartyRelationshipAttribute, ...), so I think it is extremely important to use the terms consistently and not to introduce new flavors, such as 'own Attribute', when there is already a term within our nomenclature which is suitable to use. Otherwise, I'd be concerned that the reader has no chance of keeping up if we mix technically clearly defined terms with normal linguistic expressions. But of course that doesn't need to be discussed here, so it's fine with me for now and I'd like to discuss this with you face-to-face. 👍